### PR TITLE
Expiration date: You can now teleport bread

### PR DESCRIPTION
**Pull request by Adrer**
## What Does This PR Do
Allows you to teleport bread.

## Why It's Good For The Game
We couldn't teleport bread before.

## Images of changes
https://user-images.githubusercontent.com/19361017/229228466-11cc8b98-0763-4b5c-b8ca-319defb2a96b.mp4

## Testing
I have done nothing but teleport bread for three days.

## Changelog
:cl: Adrer, ATP-Engineer
fix: Bread can now correctly be teleported.
/:cl:

### CODE REVIEW

File: code/datums/helper_datums/teleport.dm [hunk 1]
```diff
@@ -114,6 +114,21 @@
 			playsound(location,sound,60,1)
 	return
 
+/datum/teleport/proc/teleportBread()
+	var/list/food = teleatom.search_contents_for(/obj/item/reagent_containers/food)
+	if(istype(teleatom, /obj/item/reagent_containers/food))
+		food += teleatom
+	if(food.len)
+		for(var/obj/item/reagent_containers/food/fooditem in food)
+			if(fooditem.bread == TRUE)
```
Comment (AffectedArc07) [inline code/datums/helper_datums/teleport.dm:123]:
```suggestion
			if(fooditem.bread)
```

**Comment by Carthusia** [comment]
God, this should just remain a thing even after april fools

**Comment by ATP-Engineer** [comment]
I mean nobody wanted to flesh it out as an actual hazard that occurs (also people can just farm-spawn these with enough bread)

**Comment by Carthusia** [comment]
> I mean nobody wanted to flesh it out as an actual hazard that occurs (also people can just farm-spawn these with enough bread)

True, but long as it loggs what happened, who teleported bread and that the bread is hostile to everyone, but especially the person who teleported it, it would be a threat on par with that of killer tomatos, a fun reference that can be used for antag shennanigans

**Review by farie82 — APPROVED** [review]
This hurts

**Pull request merged**